packages/manager/modules/pci/src/projects/project/storages/components/user-credential-banner/template.html

```diff
@@ -1,10 +1,19 @@
 <oui-message type="success" dismissable>
-
     <!--Description-->
     <p
+        ng-if="!$ctrl.containerName"
         data-translate="{{:: $ctrl.isNewUser ? 'pci_projects_project_storages_containers_add_create_user_success_message' : 'pci_projects_project_storages_containers_add_linked_user_success_message'}}"
         data-translate-values="{ username: '<strong>'+$ctrl.user.username+'</strong>' }"
     ></p>
+    <p
+        ng-if="$ctrl.containerName"
```

Comment:
```suggestion
        data-ng-if="$ctrl.containerName"
```

Reply:
Question: in this case, `ng-if` is not a data attribute. So why should we use the data-prefix ?

Reply:
@brofik: Its an Angular.js specific attribute.  It does not have any effect as such with respect to native HTML element.

Reply:
@anooparveti 
Yes I agree that `ng-if` is an AngularJS specific attribute. 
However `data-` is an HTML specific prefix that has a semantic use to facilitate the identification of data content (text, media, img, ...) . 
So my question is : 
Why are we enforcing the use of the `data-` on an AngularJs tag that is not a data attribute ?